src/tests/dfio/SConscript

```diff
@@ -0,0 +1,29 @@
+"""Build dfio plugin"""
+import os
+
+def scons():
+    """Execute build"""
+
+    Import('env')
+
+    dfio_libs = ['daos_common', 'daos', 'dfs', 'gurt', 'uuid', 'dl']
+
+    denv = env.Clone()
+
+    if denv.subst("$FIO_SRC") != "":
```

Comment:
this still won't work until you make the change in scons_local to make that option sticky.
Jeff would know more, but you need something like:
self.add_opts(('FIO_SRC', ...) in base.py

Reply:
@jolivier23
Although FIO_SRC is set to _build.external/fio in my local branch, but I could not find the code that sets it.

Reply:
as i said you need to add it to scons_local